scrapy/utils/reqser.py

```diff
@@ -20,6 +19,7 @@ def request_to_dict(request, spider=None):
     if callable(eb):
         eb = _find_method(spider, eb)
     d = {
+        '__class__': request.__class__,
```

Comment:
I think it make sense not to store default Request class - store request class only if it is not default: 

1) it would allow to save some memory and CPU;
2) by supporting queues without `__class__` key we're making this PR compatible with disk queues created by previous Scrapy versions.